qui/tray/disk_space.py

```diff
@@ -30,11 +31,14 @@ def check_usage(self):
         if not hasattr(self.vm, 'template'):
             volumes_to_check.append('root')
         for volume_name in volumes_to_check:
-            if volume_name in self.vm.volumes:
-                size = self.vm.volumes[volume_name].size
-                usage = self.vm.volumes[volume_name].usage
-                if size > 0 and usage / size > WARN_LEVEL:
-                    self.problem_volumes[volume_name] = usage / size
+            try:
+                if volume_name in self.vm.volumes:
+                    size = self.vm.volumes[volume_name].size
+                    usage = self.vm.volumes[volume_name].usage
+                    if size > 0 and usage / size > WARN_LEVEL:
+                        self.problem_volumes[volume_name] = usage / size
+            except exc.QubesPropertyAccessError:
```

Comment:
While accessing volume properties this indeed will raise `QubesPropertyAccessError` (which inherits from `QubesDaemonAccessError`), just failed listing will raise base `QubesDaemonAccessError`.